packages/core/src/index.ts

```diff
@@ -31,7 +31,7 @@ import {
   prettyPrintExpr,
 } from "./utils/OtherUtils";
 import { bBoxDims, ops, toSvgPaintProperty } from "./utils/Util";
-import { Canvas } from "./renderer/ShapeDef";
+import { Canvas } from "./shapes/Samplers";
 import { showMutations } from "./synthesis/Mutation";
 import { getListOfStagedStates } from "./renderer/Staging";
 import { shapeAutodiffToNumber } from "engine/EngineUtils";
```

Comment:
could you also add 
```
export { shapedefs } from "./shapes/Shapes";
```
or whatever the current location is, so externals can access?

Reply:
Done!